sdk/servicebus/azure-messaging-servicebus-track1-perf/CHANGELOG.md

```diff
@@ -1,3 +1,3 @@
 # Release History
 
-## 1.0.0-beta.1 (Unreleased)
+## 1.0.0-beta.1 (2021-01-15)
```

Comment:
Do we release these perf libs too?

Reply:
No we do not release perf libraries.

Reply:
Shouldn't need to update the date here and any libraries that won't be released